guide/.vuepress/components/Definition.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div class="card">
+		<span class="title"> Info </span>
+		<span class="content">{{ content }}</span>
+	</div>
+</template>
+
+<script setup lang="ts">
+const props = defineProps(['title', 'content']);
+</script>
+
+<style setup lang="scss">
+.card {
+	border-radius: 0.375rem;
+	border-color: #7289da;
+	border-radius: 0.375rem;
+	padding: 0.1rem 1.5rem;
+	border-left-width: 0.5rem;
+	border-left-style: solid;
+	margin: 1rem 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	background-color: #23272a;
+	color: #fff;
+	padding: 1rem;
+
+	// someone is gonna scream at me for this i can sense it
+	&:hover {
+		box-shadow: 0 0 0.5rem 0.5rem rgba(114, 137, 218, 0.5);
+		transition: box-shadow 0.2s ease-in-out;
+		transition-timing-function: cubic-bezier(0.175, 0.885, 0.32, 1.275);
+	}
+	&:hover ~ .card {
+		opacity: 0;
+		transition: opacity 0.2s ease-in-out;
+		transition-timing-function: cubic-bezier(0.175, 0.885, 0.32, 1.275);
+	}
```

Comment:
Truly, this styling makes it inconsistent with the Tip/Warning/Danger boxes.

Reply:
Yeah, I knew that was going to be an issue, it just looked cool IMO. 829e94a77cd9a4ece76eaba9f45dd0b6917ba76d